.yarnrc.yml

```diff
@@ -0,0 +1,11 @@
+enableGlobalCache: true
+
+httpTimeout: 300000
+
+nodeLinker: node-modules
+
+npmRegistryServer: 'https://registry.npmjs.org/'
+
+compressionLevel: 0
+
+yarnPath: .yarn/releases/yarn-4.5.0.cjs
```

Comment:
Same for this file, could we use some default that `yarn` generates here?  E.g. `enableGlobalCache` is true by default anyways: https://yarnpkg.com/advanced/changelog#major-changes

Reply:
I had a copy from the other repo we used as inspiration to fix that ugly issue yesterday. I replaced with the same as in the upgrade-helper for backstage: https://raw.githubusercontent.com/backstage/upgrade-helper-diff/release/0.5.20/.yarnrc.yml